openhtf/io/frontend/src/app/station.service.ts

```diff
@@ -97,12 +97,12 @@ export class StationService extends SubscriptionService {
    */
   onmessage(msg) {
     this.reachable = true;
-    if (!msg) {
+    let parsedData = JSON.parse(msg.data);
```

Comment:
there's no checking on the format of msg here, or that it's not null, might be worth a descriptive console log message?